ArduCopter/mode.h

```diff
@@ -1513,10 +1515,16 @@ class ModeTurtle : public Mode {
     bool allows_arming(AP_Arming::Method method) const override;
     bool is_autopilot() const override { return false; }
     void change_motor_direction(bool reverse);
+    bool has_direct_motor_output() const override { return true; }
+    void output_to_motors() override;
```

Comment:
I think it makes sense to have a default implementation and we always call the flight_mode->output_to_motors() call.

Reply:
Fixed